File: c++/Ram.cpp
#include <iostream>
#include <string>

using namespace std;

class Ram
{
    private:
        /* atribute */
        int capacity;
        int price;
    public:
        Ram(){
        }

        Ram(int c, int p){
            capacity = c;
            price = p;
        }

        //setter
        void setRamCapacity(int c){
            capacity = c;
        }

        void setRamPrice(int p){
            price = p;
        }

        //getter
        int getRamCapacity(){
            return capacity;
        }

        int getRamPrice(){
            return price;
        }

        //print atribute
        void showRam(){
            cout << "Ram Capacity    : " << capacity << " Gb" << endl;
            cout << "Ram Price       : Rp. " << price << endl;
        }

        ~Ram(){}
};

File: php/Ram.php
<?php

class Ram
{
    private $price = 0;
    private $capacity = 0;

    # Constructor
    public function __construct($c=0, $p=0)
    {
        $this->price = $p;
        $this->capacity = $c;
    }

    # Setter and Getter
    public function setPrice($p=0)
    {
        $this->price = $p;
    }

    public function getPrice()
    {
        return $this->price;
    }

    public function setCapacity($c=0)
    {
        $this->capacity = $c;
    }

    public function getCapacity()
    {
        return $this->capacity;
    }

    # Menampilkan isi kelas
    public function showRam()
    {
        echo "Ram  Capacity   : " . $this->capacity . " Gb<br>";
        echo "Ram Price      : Rp. " . $this->price . "<br>";
    }
}

?>